CMakeLists.txt

```diff
@@ -240,7 +240,9 @@ endif()
 if("${ENABLE_ALL_WARNINGS}")
   if(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
     set(KNOWN_COMPILER_SPECIFIC_WARNINGS
-        "-Wno-range-loop-analysis -Wno-mismatched-tags")
+        "-Wno-range-loop-analysis \
+         -Wno-mismatched-tags \
+         -Wno-nullability-completeness")
```

Comment:
Shall we also consider adding one more flag `-Wdeprecated-declarations` to suppress issues like #3317?
I raised this suppression because `sprintf` is only deprecated in MacOS. There are still a lot of dependency libraries (such as duckdb) using this well-known api.

Reply:
I would keep it like this for now.  If this becomes unmanageable in the future we can consider add it here.